website/docs/gen-ai/inference/Llama2.md

```diff
@@ -0,0 +1,310 @@
+
+![Ray Dashboard](img/ray-dashboard.png)
+
+### To Test the Llama-2-Chat Model
+Once you see the status of the model deployment is in `running` state then you can start using Llama-2-chat.
+
+You can use the following URL with a query added at the end of the URL.
+
+    http://<NLB_DNS_NAME>/serve/infer?sentence=what is data parallelism and tensor parallelisma and the diffrences
+
+You will see an output like this in your browser:
+
+![Chat Output](img/llama-2-chat-ouput.png)
+
+## Deploying the Gradio WebUI App
+Discover how to create a user-friendly chat interface using [Gradio](https://www.gradio.app/) that integrates seamlessly with deployed models.
+
+Let's deploy Gradio app locally on your machine to interact with the LLama-2-Chat model deployed using RayServe.
+
+:::info
+
+The Gradio app interacts with the locally exposed service created solely for the demonstration. Alternatively, you can deploy the Gradio app on EKS as a Pod with Ingress and Load Balancer for wider accessibility.
+
+:::
+
+### Execute Port Forward to the Llama-2 Ray Service
+First, execute a port forward to the Llama-2 Ray Service using kubectl:
+
+```bash
+kubectl port-forward svc/Llama-2-service 8000:8000 -n Llama-2
+```
+
+### Deploy Gradio WebUI Locally
+
+#### Create a Virtual Environment
+Create a Python virtual environment in your machine for the Gradio application:
+
+```bash
+cd ai-ml/trainium-inferentia/examples/gradio-ui
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+#### Install Gradio ChatBot app
+Install all the Gradio WebUI app dependencies with pip
+
+```bash
+pip install gradio requests
+```
+
+#### Invoke the WebUI
+Run the Gradio WebUI using the following command:
+
+NOTE: `gradio-app.py` refers to the port forward url. e.g., `service_name = "http://localhost:8000" `
+
+```bash
+python gradio-app.py
+```
+
+You should see output similar to the following:
+
+```text
+Using cache from ~/data-on-eks/ai-ml/trainium-inferentia/examples/gradio-ui/gradio_cached_examples/16' directory. If method or examples have changed since last caching, delete this folder to clear cache.
+
+Running on local URL:  http://127.0.0.1:7860
+
+To create a public link, set `share=True` in `launch()`.
+```
+
+#### 2.4. Access the WebUI from Your Browser
+Open your web browser and access the Gradio WebUI by navigating to the following URL:
+
+http://127.0.0.1:7860
+
+You should now be able to interact with the Gradio application from your local machine.
+
+![Gradio Llama-2 AI Chat](img/gradio-llama-ai-chat.png)
+
+## Conclusion
+In conclusion, you will have successfully deployed the **Llama-2-13b chat** model on EKS with Ray Serve and created a chatGPT-style chat web UI using Gradio.
+This opens up exciting possibilities for natural language processing and chatbot development.
+
+In summary, when it comes to deploying and scaling Llama-2, AWS Neuron instances offer a compelling advantage.
```

Comment:
as above - AWS Neuron is the SDK. We prefer to call-out trn1/inf2 instances (powered by AWS Trainium and Inferentia) rather than brand the instances as 'Neuron'.